cli_reference/get_started_cli.adoc

```diff
@@ -93,9 +115,15 @@ users:
 ----
 ====
 
-Configuration files can be used for example to configure OpenShift to support the setup of multiple servers, namespaces, and users so that you can switch easily between them. Multiple configuration files are allowed; they are loaded at runtime and merged together along with override options specified from the command line. 
+Configuration files can be used for example to configure OpenShift to support
+the setup of multiple servers, namespaces, and users so that you can switch
+easily between them. Multiple configuration files are allowed; they are loaded
+at runtime and merged together along with override options specified from the
+command line.
 
-There are a few ways to provide `osc` a configuration file. Basically when loading configurations the following priority rules for configuration file loading are observed (in this exact order):
+There are a few ways to provide `osc` a configuration file. Basically when
+loading configurations the following priority rules for configuration file
+loading are observed (in this exact order):
 
```

Comment:
[Suggestion]
There are multiple methods that you can use to create a configuration file to be used with `osc`. The following rules are  observed in order when a configuration file is loaded: